ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/LedgerTestCasesRunner.scala

```diff
@@ -75,12 +75,17 @@ final class LedgerTestCasesRunner(
       session
         .createTestContext(testName, identifierSuffix)
         .flatMap { context =>
-          val start = System.nanoTime()
-          val result = test(context).map(_ => Duration.fromNanos(System.nanoTime() - start))
-          logger.info(
-            s"Started '${test.description}'${test.repetition.fold("")(r => s" (${r._1}/${r._2})")} with a timeout of $scaledTimeout."
-          )
-          result
+          val features = context.configuredParticipants.head.features
```

Comment:
Huh, I thought there has already been support for that here.

Reply:
Unfortunately not, the features are associated to a session/participant so we have to just choose one to get the features